certified-connectors/CiviQ/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2868 @@
+                        },
+                        "description": "Consultation Submissions"
+                      }
+                    }
+                  },
+                  "description": "Consultations"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "type": "string",
+              "description": "Could not find resource"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "summary": "Consultations List",
+        "operationId": "Consultations",
+        "parameters": [
+          {
+            "name": "status",
+            "x-ms-summary": "Consultation Status",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "all",
+              "closed",
+              "upcoming",
+              "open"
+            ],
+            "default": "open",
+            "description": "Consultation Status"
+          },
+          {
+            "name": "access",
+            "x-ms-summary": "Consultation Access",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "all",
+              "public",
+              "private"
+            ],
+            "description": "Consultation Access"
+          }
+        ],
+        "description": "Operation to retrieve from the portal the list of Consultations.",
+        "tags": [
+          "Consultations"
+        ]
+      }
+    },
+    "/consultations/{ConsultationID}.json": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Consultation data",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "Consultation-Schema",
+                "parameters": {
+                  "ConsultationID": {
+                    "parameter": "ConsultationID"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "type": "string",
+              "description": "Could not find resource"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "summary": "Consultation by ID",
+        "operationId": "Consultation-id",
```

Comment:
@RicardoMartins1993 all operationIds should be in Camel case. please change to "ConsultationId". Please fix all operationIds in your swagger file.

Reply:
@Amjed-Ayoub  Done